content/cli/smartmodules/hub.md

```diff
@@ -73,9 +76,18 @@ infinyon/regex-filter2@0.1.0
 
 SmartModule Packages are listed in the hub as in the form of `<group>/<package name>@version`. 
 
-Group `infinyon` packages are published by InfinyOn, the maker of Fluvio.  Third-party developers may also upload public packages which appear in other groups. The Hub service enforces signing and publishing restrictions on uploaded packages. Once a signed package of given group has been published, only the original signing key will be accepted for other packages in the same group.
+Group `infinyon` packages are published by InfinyOn, the maker of Fluvio.  Third-party developers may also upload public packages which appear in other groups. The Hub service enforces signing and publishing restrictions on uploaded packages. Once a signed package of given group has been published, only the original owner of that package will be accepted into the same group.
```

Comment:
I don't quite understand this sentence:

```
 Once a signed package of given group has been published, only the original owner of that package will be accepted into the same group.
```

Reply:
@digikata I approved, then I noticed that some of the items are still unresolved.

Reply:
No problem, the final approval is still blocked on the comments.  I'll update for the commands and you'll have a chance to review again

Reply:
updated!